src/modules/transaction/store/actions.js

```diff
@@ -96,9 +96,10 @@ export const transactionBroadcasted =
     const moduleCommand = joinModuleAndCommand(transaction);
     const paramsSchema = moduleCommandSchemas[moduleCommand];
     let broadcastResult;
-    const dryRunResult = await dryRun({ transaction, serviceUrl, paramsSchema });
 
-    if (dryRunResult.data?.result === 1) {
+    const [error, dryRunResult] = await to(dryRun({ transaction, serviceUrl, paramsSchema }));
+
+    if (dryRunResult?.data?.result === 1) {
```

Comment:
I think it would be best if we have an enum of dryrun results instead of using magic values here

e.g

```
const dryRunResultTypes = {
  ERROR_1: 1,  // ERROR_1 is just a mock string
  ERROR_0: 0, // ERROR_0 is just a mock string.
}
```

Then consumption can be used as 

```suggestion
    if (dryRunResult?.data?.result === dryRunResultTypes.ERROR_1) {
```